media/js/base/stub-attribution.js

```diff
@@ -326,7 +326,7 @@ if (typeof window.Mozilla === 'undefined') {
      * Gets the client ID from the GA object.
      * @returns {String} client ID.
      */
-    StubAttribution.getGAVisitID = function () {
+    StubAttribution.getGAVisitorID = function () {
```

Comment:
For consistency, I'd rename this method to `getGAClientID` and use `clientID` everywhere (except when we pass this value to the stub service since the field must be named `visit_id` for now)

Reply:
I went back and forth on this as to what's best. If you think it makes sense to differentiate here, that works for me

Reply:
Yeah, I'd rather use the "correct" name as much as possible, and document why we need to use `visit_id` in the attribution data (e.g. in a code comment).

And maybe this is something we could improve actually, maybe the stub service could accept `client_id` instead of `visit_id` so that bedrock uses the correct name everywhere, consistently. Then, in the stub service, we'd still use `visit_id` where it is required (i.e. in the log statements that are forwarded to bigquery/redash).

Reply:
That makes sense, I can make that change here so we pass `client_id` if that's something you could add easily enough

Reply:
I filed https://github.com/mozilla-services/stubattribution/issues/153